Add note about BQ access requests (#708)
mozilla/data-docs: src/cookbooks/bigquery/access.md

## BigQuery Access Request

> > **⚠**  If you are a member of certain staff groups such as Data Engineering or Data Science you do not need to request BigQuery access specifically; it is automatically granted.

Reviewer: "_certain staff groups such as..._" indicates there are other groups besides Data Engineering and Data Science that are automatically granted BigQuery access, which might leave non-DE/DS people wondering if they're a member of those other groups.  I'd suggest either listing all such groups, or making the statement more definitive about the DE & DS groups, like:
```suggestion
> **⚠**  If you are a member of the Data Engineering or Data Science staff groups you do not need to request BigQuery access specifically; it is automatically granted.
```

Also, is there any way for people to empirically check whether they're a member of those staff groups?

Author: I left this vague because there are other groups besides Data Engineering and Data Science that are automatically granted BigQuery access, but keeping the list up to date here is not reasonable. There is a way to check whether you're a member of a staff group that has access to mozilla-confidential data, which is by querying mozilla-confidential data from the mozdata project. The flow that would cut down on extraneous requests the most would include telling people to check whether they can access data before filing a request.

Given there are plans to redo permissions to obviate the vast majority of requests via https://mozilla-hub.atlassian.net/browse/DSRE-544 anyway, I think the most reasonable thing to do here is to make the statement definitive as to DE and DS and remove this entire section when DSRE-544 is implemented.
